### PR TITLE
DOCSP-5746 update for Data Lake limitations

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
JIRA: https://jira.mongodb.org/browse/DOCSP-5746
Stage Links:

https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/collections.html
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/validation.html
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/indexes.html
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/query-plan.html
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/documents/insert.html#limitation
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/import-export.html
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/databases.html
https://docs-mongodbcom-staging.corp.mongodb.com/compass/kanchanasekhar/DOCSP-5746/performance.html

### CODE REVIEW

File: source/validation.txt [hunk 1]
```diff
@@ -37,6 +28,20 @@ specific shape or only allowing a specified range of values in fields.
...
 
+Limitations
+-----------
+
+The :guilabel:`Validation` tab is not available: 
+
+- In :guilabel:`Compass Community Edition`. 
+
+- If you are connected to a 
+  `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/validation.txt]: We have a replacement for links to the atlas docs. You can do:

```
:atlas:`Data Lake </data-lake>`
```

File: source/query-plan.txt [hunk 2]
```diff
@@ -21,6 +21,12 @@ on the left hand pane, then click :guilabel:`Explain Plan`.
 .. figure:: /images/compass/query-plan.png
   :figwidth: 815px
 
+Limitation
+----------
+
+The :guilabel:`Explain Plan` tab is not available if you are connected 
+to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/query-plan.txt]: See comment about replacement.

File: source/performance.txt [hunk 3]
```diff
@@ -10,7 +10,14 @@ Performance
...
 
-*Not Available in Compass Community Edition*
+Limitations
+-----------
+
+The :guilabel:`Performance` tab is not available:
+
+- In Compass Community Edition.
+
+- If you are connected to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/performance.txt]: See comment about replacement.

File: source/indexes.txt [hunk 4]
```diff
@@ -12,7 +12,13 @@ Indexes Tab
...
    :class: singlecol
 
-.. include:: /includes/extracts/readonly-not-permitted-indexes.rst
+Limitations
+-----------
+
+- .. include:: /includes/extracts/readonly-not-permitted-indexes.rst
+
+- The :guilabel:`Indexes` tab is not available if you are connected 
+  to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/indexes.txt]: See comment about replacement.

File: source/import-export.txt [hunk 5]
```diff
@@ -12,7 +12,13 @@ Import and Export Data
...
    :class: singlecol
 
-.. include:: /includes/extracts/readonly-not-permitted-import.rst
+Limitations
+-----------
+
+- .. include:: /includes/extracts/readonly-not-permitted-import.rst
+
+- Import data is not available if you are 
+  connected to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/import-export.txt]: See comment about replacement.

File: source/documents/insert.txt [hunk 6]
```diff
@@ -52,6 +52,12 @@ field name and value.
           "phone": "555-555-1212"
         }
 
+Limitation
+----------
+
+The :guilabel:`Insert Document` button is not available if you are connected 
+to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/documents/insert.txt]: See comment about replacement.

File: source/databases.txt [hunk 7]
```diff
@@ -12,7 +12,13 @@ Databases
...
    :class: singlecol
 
-.. include:: /includes/extracts/readonly-not-permitted-databases.rst
+Limitations
+-----------
+
+- .. include:: /includes/extracts/readonly-not-permitted-databases.rst
+
+- The :guilabel:`Create Database` button is not available if you are connected 
+  to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/databases.txt]: See comment about replacement.

File: source/collections.txt [hunk 8]
```diff
@@ -12,7 +12,13 @@ Collections and Views
...
    :class: singlecol
 
-.. include:: /includes/extracts/readonly-not-permitted-collections.rst
+Limitations
+-----------
+
+- .. include:: /includes/extracts/readonly-not-permitted-collections.rst
+
+- The :guilabel:`Create Collection` button is not available if you are connected 
+  to a `Data Lake <https://www.mongodb.com/atlas/data-lake>`__.
```
Comment (jeff-allen-mongo) [inline source/collections.txt]: See comment about replacement.

File: source/indexes.txt [hunk 9]
```diff
@@ -49,6 +55,11 @@ For each index, Compass displays the following information:
      - Any special properties (such as uniqueness, partial) of the
        index.
 
+Limitation
```
[the remark this answers is not in the document]
Comment (kanchana-mongodb) [reply source/indexes.txt]: @jeff-allen-mongo fixed; please see stage

File: source/collections.txt [hunk 10]
```diff
@@ -12,7 +12,13 @@ Collections and Views
    :depth: 1
    :class: singlecol
 
-.. include:: /includes/extracts/readonly-not-permitted-collections.rst
+Limitations
```
[the remark this answers is not in the document]
Comment (kanchana-mongodb) [reply source/collections.txt]: @jeff-allen-mongo fixed; please see stage

File: source/validation.txt (hunk 1 above)
Comment (kanchana-mongodb) [reply source/validation.txt]: @jeff-allen-mongo fixed; please see stage

File: source/query-plan.txt (hunk 2 above)
Comment (kanchana-mongodb) [reply source/query-plan.txt]: @jeff-allen-mongo fixed; please see stage

File: source/performance.txt (hunk 3 above)
Comment (kanchana-mongodb) [reply source/performance.txt]: @jeff-allen-mongo fixed; please see stage

File: source/indexes.txt (hunk 4 above)
Comment (kanchana-mongodb) [reply source/indexes.txt]: @jeff-allen-mongo fixed; please see stage

File: source/import-export.txt (hunk 5 above)
Comment (kanchana-mongodb) [reply source/import-export.txt]: @jeff-allen-mongo fixed; please see stage

File: source/documents/insert.txt (hunk 6 above)
Comment (kanchana-mongodb) [reply source/documents/insert.txt]: @jeff-allen-mongo fixed; please see stage

File: source/databases.txt (hunk 7 above)
Comment (kanchana-mongodb) [reply source/databases.txt]: @jeff-allen-mongo fixed; please see stage

File: source/collections.txt (hunk 8 above)
Comment (kanchana-mongodb) [reply source/collections.txt]: @jeff-allen-mongo fixed; please see stage

File: source/documents/insert.txt [hunk 11]
```diff
@@ -52,6 +52,12 @@ field name and value.
           "phone": "555-555-1212"
         }
 
+Limitation
+----------
+
+The :guilabel:`Insert Document` button is not available if you are connected 
+to a :atlas:`Data lake </data-lake>`.
```
Comment (jdestefano-mongo) [inline source/documents/insert.txt]: @kanchana-mongodb Can you submit another PR to capitalize "lake" ?

"Data Lake"
